src/main.c

```diff
@@ -81,9 +131,14 @@ static void init_bk(int argc, char *argv[])
 
 static void run_application()
 {
+    if (strlen(bw.mix_cmd) == 0) {
```

Comment:
This is the line that segfaults. Change it to `bw.mix_cmd == NULL`. I like this feature, but I had questions on how we support it that I wanted to get your opinion on before changing anything. See the other comment.